backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanConfig.kt

```diff
@@ -145,6 +145,24 @@ class KonanConfig(val project: Project, val configuration: CompilerConfiguration
         getIncludedLibraries(includedLibraryFiles, configuration, resolvedLibraries)
     }
 
+    val usedDynamicCaches: List<String>
+        get() = configuration.get(KonanConfigKeys.USED_DYNAMIC_CACHES)!!
+
+    val usedStaticCaches: List<String>
+        get() = configuration.get(KonanConfigKeys.USED_STATIC_CACHES)!!
+
+    // TODO: is "featured libraries" concept applicable here?
+    //   cache may contain more libraries than required,
```

Comment:
Find mapping "one klib -> one cache element` more intuitive, and thus - "cache may contain more libraries than required" shalln't happen, as we pass exactly what we willing to compile.

Reply:
> Find mapping "one klib -> one cache element` more intuitive

I find this too restrictive. I believe compiler cache support should be more flexible.
For example, in some cases using cache may require user to adjust some build configuration (e.g. embed dynamic cache to application bundle on Apple platforms). Using single cache binary in this case may be easier than using binary per klib.